src/test/java/com/fasterxml/jackson/databind/ser/filter/JsonValueIgnore3647Test.java

```diff
@@ -0,0 +1,102 @@
+package com.fasterxml.jackson.databind.ser.filter;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.databind.BaseMapTest;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+// [databind#3647] : Support @JsonIgnoreProperties to work with @JsonValue
+public class JsonValueIgnore3647Test extends BaseMapTest
+{
+    static class Foo3647 {
```

Comment:
@JooHyukKim Making this `final` will make test fail. I think this is because `_valueSerializer` has then been pre-fetched.... so fix here won't work for all cases.